cms/src/test/resources/test-data/api-specifications/SwaggerCodeGenOpenApiSpecificationJsonToHtmlConverterTest/customised-codegen-v3-generated-spec.html

```diff
@@ -2133,14 +2227,103 @@ <h3>Body</h3>
                                                     <div class="httpparams__required">Required</div>
                                                         
                                                         
+                                                                <h5 class="body__contenttype">
```

Comment:
Similarly to the previous comment: is it definitely correct that this block of content is added here?

Reply:
Fixed